Topbar (independent of login data) PR

Bence021 opened the pull request:
Features for Issue #6 

Most of the comments help learning tailwind css styling.

handleLogout &  handleSettings functions will be developt later on.
handleSidebarOpen will be developt with sidebar feature.

There was a false warning, so my solution was to turn of the Unknown at-rule.

> I run into this also:
> `Error: Failed to load plugin 'next/hooks'`
> 
> My solution was to:
> `npm install next@latest eslint-config-next@latest`
> 
> To this solution I didn't upload the diff files, maybe it's just a local error. Idk
> 

Some pictures of the implementation:

<img width="531" height="368" alt="image" src="https://github.com/user-attachments/assets/0b3116b9-e71b-4df7-bae8-db8a3c5b6c6c" />

<img width="288" height="572" alt="image" src="https://github.com/user-attachments/assets/8563850b-ab2b-48f4-8844-80a49761ec44" />

Bence021 committed:
Topbar & topbar usage in chat page (independent of login data) #6

balog-b reviewed (changes requested):
`absolute` should be used to place elements freely on another existing element, that would otherwise not be possible. Here, I don't think you should use it. The reason is if you would want to add a new element to the topbar in the future, you would need to place it absolutely aswell, which may not be that easy.

I advise that you place 2 flexboxes on the topbar. One uses `justify-start` (default, so it can be ommited) and the other one uses `justify-end`. Put padding and spacing on it and you can achieve the same look, and can add new elements to the topbar easily, without having to worry about the exact place of the elements.

Alternatively, you can also create just 1 flexbox for the items on the left, and add a div with `flex-1` to it. This will cause the box to fill out the whole space available, and push the user button to the side, so it stays on the right.

Bence021 committed:
fix absolute positioning and modified the appearance a little #6

Bence021 commented:
@balog-b I understand what you meant and I agree with it. That's why I changed it according to the suggestions you gave, with 3 flexboxes. I also modified the appearance a little.

Bence021 committed:
fix width (due to later arrangement)

balog-b reviewed (approved):
Looks good now. Thank you 🙏
